src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -52,160 +58,160 @@
       <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.EnumWindows.cs">
-      <Link>Common\Interop\Windows\Interop.EnumWindows.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.EnumWindows.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.GetWindowTextLength.cs">
-      <Link>Common\Interop\Windows\Interop.GetWindowTextLength.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.GetWindowTextLengthW.cs">
+      <Link>Common\Interop\Windows\user32\Interop.GetWindowTextLengthW.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.SendMessageTimeout.cs">
-      <Link>Common\Interop\Windows\Interop.SendMessageTimeout.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.SendMessageTimeout.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.GetWindowText.cs">
-      <Link>Common\Interop\Windows\Interop.GetWindowText.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.GetWindowTextW.cs">
+      <Link>Common\Interop\Windows\user32\Interop.GetWindowTextW.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.GetWindow.cs">
-      <Link>Common\Interop\Windows\Interop.GetWindow.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.GetWindow.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.PostMessage.cs">
-      <Link>Common\Interop\Windows\Interop.PostMessage.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.PostMessage.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.GetWindowLong.cs">
-      <Link>Common\Interop\Windows\Interop.GetWindowLong.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.GetWindowLong.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.WaitForInputIdle.cs">
-      <Link>Common\Interop\Windows\Interop.WaitForInputIdle.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.WaitForInputIdle.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.IsWindowVisible.cs">
-      <Link>Common\Interop\Windows\Interop.IsWindowVisible.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.IsWindowVisible.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.GetWindowThreadProcessId.cs">
-      <Link>Common\Interop\Windows\Interop.GetWindowThreadProcessId.cs</Link>
+      <Link>Common\Interop\Windows\user32\Interop.GetWindowThreadProcessId.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.CloseHandle.cs">
       <Link>Common\Interop\Windows\Interop.CloseHandle.cs</Link>
```

Comment:
We're adding a subfolder in the Link for user32, advapi32, etc., but not for kernel32?